pages/my-stream/index.tsx

```diff
@@ -8,22 +8,23 @@ const Header = dynamic(() => import("../../components/header/Header"), {
   loading: () => <HeaderPlaceholder />,
 });
 
-export default function BrainPage() {
+export default function MyStreamPage() {
   const breadcrumbs: Crumb[] = [
     { display: "Home", href: "/" },
     { display: "Brain" },
+    { display: "My Stream" },
```

Comment:
this feels a bit weird, every crumb should have be a link except the last one

in this case maybe you need to remove the "Brain" crumb